packages/design-system/src/components/Table/TableDataCell.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableDataCell = ({
+  children,
+  className,
+  stackedTitle,
+  stackedClassName,
+  type,
+  ...attributeOptions
+}) => {
+  const classes = classNames(
+    'ds-c-table__cell',
+    className,
+    type === 'numeric' ? 'ds-c-table__cell--numeric' : null
+  );
+
+  const renderStackedTitle = () => {
+    const stackedClasses = classNames(
+      'ds-c-table--stacked__col-header',
+      'ds-u-font-weight--bold',
+      stackedClassName
+    );
+    const isValidStackedTitle = stackedTitle && stackedTitle.length > 0;
+    return (
+      { isValidStackedTitle } && (
+        <span aria-hidden="true" className={stackedClasses}>
+          {stackedTitle}
+        </span>
+      )
+    );
+  };
+
+  return (
+    <td className={classes} role="cell" {...attributeOptions}>
+      {renderStackedTitle()}
+      {children}
+    </td>
+  );
+};
+
+TableDataCell.defaultProps = {
+  type: 'text',
+};
+
+TableDataCell.propTypes = {
+  /**
+   * The table data cell contents.
+   */
+  children: PropTypes.node,
+  /**
+   * Additional classes to be added to the table cell element.
+   */
+  className: PropTypes.string,
+  /**
+   * The `headers` attribute contains a list of `id` attributes of the associated data. If there is more than one id, they are separated by spaces.
+   */
+  headers: PropTypes.string,
+  /**
+   * Additional classes to be added to the stacked Title element.
+   */
+  stackedClassName: PropTypes.string,
```

Comment:
I noticed that the stackedClassName does not get applied to the spans at the row inside a `th`
Should they be applied to the span in the `th` here? 

In the example, I set the class name `stackerCell`

![Screen Shot 2020-06-15 at 11 01 18 AM](https://user-images.githubusercontent.com/1449852/84673708-11ae9d00-aef8-11ea-8735-28e175b01fc5.png)

Reply:
It applied stackedClassName to the span for me. Can you try again.

Reply:
It worked for me, thanks!